packages/console/src/pages/Applications/index.tsx

```diff
@@ -13,6 +16,28 @@ const Applications = () => {
         <CardTitle title="applications.title" subtitle="applications.subtitle" />
         <Button disabled title="admin_console.applications.create" />
       </div>
+      <table className={styles.table}>
+        <thead>
+          <tr>
+            <td className={styles.applicationName}>Application Name</td>
+            <td>Client ID</td>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              <ItemPreview
+                title="Default App"
+                subtitle="Single Page Application"
```

Comment:
Should add these title and subtitle contents to i18n phrases

Reply:
will do in the next PR

Reply:
will do soon